models/db.py

```diff
@@ -538,22 +538,17 @@ def check_for_donate_or_build(field_dict, id_of_insert):
 def admin_logger(logger):
     if settings.academy_mode:
         if auth.user:
-            sid = auth.user.username
-            course = auth.user.course_name
-        else:
```

Comment:
I don't understand this change.

What was causing the exceptions that this PR is trying to avoid?

Reply:
This attempts to insert a `useinfo` record where `useinfo.course_id == "Unknown"`. However, since `useinfo.course_id` [is a foreign key](https://github.com/bnmnetp/BookServer/blob/main/bookserver/models.py#L149) for `courses.course_name`, this violates the constraint: the course name "Unknown" does not exist in the `courses` table.

Reply:
The foreign key constraint is new.  I would rather use "boguscourse" which has always been a placeholder course than to simply lose all of this information.  This is valuable information for analyzing the use of the admin interface.

Reply:
That sounds good to me. Currently, the BookServer doesn't create `boguscourse` in `crud.create_initial_courses_users`. I assume we'll remove this function, and instead use `rsmanage initdb` instead? (I see `boguscourse` in `rsmanage.initialize_tables`).